### ISSUE TITLE
Crashing on Pause

### ISSUE THREAD

**Issue by JamesMDenton** [henbagle/LE1CommunityPatch#240]
**Describe the bug**
When pressing "esc" to pause, the game crashes.

**To Reproduce**
Fresh install of the game, install only the LE1 Community Patch. Load into the game. Press "Esc".

**Expected behavior**
When you press "Esc" the pause menu should come up instead of crashing to desktop

**Other Considerations**
_Have you considered if this feature was intended by BioWare?_ 
I seriously doubt BioWare wants to crash the game instead of allowing pausing. And I have checked, without the LE1CP you can pause just fine.

_Is this instead a QoL feature, or a potential new mod idea?_ 
No

_Would fixing this bug severely change a part of the game experience?_
It would allow people to play the game and pause.

**Screenshots & Saves**
If applicable, add screenshots or savegames to help explain the issue

**Game file**
If possible, do you know which game file this bug is located in?

**Comment by JamesMDenton** [comment]
Extra info:

I picked up a copy of LE1CP 1.5.1, and tried it with a fresh install on a completely different drive, it also had the same pause issue.

The pause issue ONLY happens with the LE1CP. 
Zero mods, no issue. 
18 other mods installed, no issue. 
ONLY LE1CP installed, pause crashes the game. 
LE1CP with 18 other mods, crashes on pause.

The save was created on an unmodded version of the game right after you are given control of Shepard on the Normandy.

If I try to start a game with only the LE1CP installed, it crashes right after the intro sequence, when joker jumps the Normandy before the dialogue scene where you can choose responses.

**Comment by JamesMDenton** [comment]
Even more info:
Installed with ME3Tweaks Mod Manager version 8.2.4. Build 133 February 25, 2024 @ 05:56.

That is by far the most recently updated element in this chain that I can find. I might see what happens using an older ME3Tweaks version.

**Comment by henbagle** [comment]
Yep, this seems to be a known issue with that version of the mod manager. I believe that version has been pulled from the servers but anyone who already updated will have to manually downgrade. The problem has been determined and a fix has been made but I believe the update is still in the progress of being rolled out. You can go to the ME3Tweaks discord for any continuing updates.

As this is an issue with the mod manager and not with the Community Patch, I will be closing this issue.
